duneanaobj/StandardRecord/StandardRecord.h

```diff
@@ -162,6 +190,8 @@ namespace caf
     float LepE;
     float LepNuAngle;
 
+    SRVector3D LepEndpoint;
```

Comment:
I'm letting all the capitalization at top-level slide, because the top level of the CAF is a mess in any case.